covid-projections/covid-projections
# update the email unsubscribe form to use regions

Updates the `AlertUnsubscribe` component to work with regions (it's broken in the feature branch) to use regions instead of locations.

### Testing
- Go to `http://localhost:3000/alert_unsubscribe?email=pnavarrc@gmail.com` (the component doesn't work without the `email` param)
- You should see your alert subscriptions
- Remove some locations, unsubscribe
